### PR TITLE
Feature/add node addon to common package

### PR DESCRIPTION
**Pull request by almilosz**
### Details:
 - implement ITensor, IShape and loadModel() function in common package using node addon binding

### Tickets:
 - 107640

### CODE REVIEW

File: src/bindings/js/node/lib/index.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const ov = require('../build/Release/ov_node_addon.node');
```
Comment (vishniakov-nikolai) [inline src/bindings/js/node/lib/index.ts:1]:
```suggestion
const ov: ovNodeModule = require('../build/Release/ov_node_addon.node');
```

**Review by vishniakov-nikolai — APPROVED** [review]
Everything is correct.
Please, merge latest version of **js-wasm-proto** to your branch and adapt precision in node package

File: src/bindings/js/node/lib/index.ts (hunk 1 above)
Comment (almilosz) [reply src/bindings/js/node/lib/index.ts]: I also had to import ovNodeModule:
`import loadModel, { ovNodeModule } from './node-model';`

File: samples/js/node/README.md [hunk 2]
```diff
@@ -20,3 +20,14 @@ To run ppp_example.js you have to additionally specify a path to the `resnet50-v
 $ npm install
 $ node <example-name> <path-to-model> [path-to-image]
 ```
+
+## Common API example 
+1. Install OpenVINO™ Runtime on your system and export its path.
+2. From *openvino/src/bindings/js/common* run: `npm i` to compiled common package files
+3. From *openvino/src/bindings/js/node* run: `npm i` to build Node addon package
+4. Then from *openvino/src/bindings/js/node* run `npx tsc` to compile common node api package
```
Comment (vishniakov-nikolai) [inline samples/js/node/README.md:28]: Need to move `postinstall` part

**Pull request merged**